release: 10.0.0 (#3526)
rome/tools: npm/rome/README.md

## Installation

```shell
npm install --save-dev rome 
```

## Usage

Start by creating an **optional** configuration file:

```shell

> npx rome init

Reviewer: We should either use `npx` everwhere or only use `rome` (I think this is preferred)

```

Format files:

```shell
rome format --write ./path ./path/to/file.js
```

Run the linter:

```shell
rome check ./path ./path/to/file.js
```

## Documentation

> For a complete documentation, please visit the [official website].

Reviewer: Link?

Reviewer: I don't think it's worth maintaining two documentations. I recommend removing the whole section and linking to the documentation on our website.

Contribution and development instructions can be found in [CONTRIBUTING](./CONTRIBUTING.md).

Additional project coordination and real-time discussion happens on our [Discord server](https://discord.gg/rome). 

> Remember that all activity on the Discord server is still moderated and will be strictly enforced under the project's [Code of Conduct](./CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
Remember that all activity on the Discord server is moderated according to the project's [Code of Conduct](./CODE_OF_CONDUCT.md).
```
